src/main/java/muramasa/gti/GregTech.java

```diff
@@ -12,24 +12,36 @@
 import muramasa.gti.loader.MaterialRecipeLoader;
 import muramasa.gti.loader.WorldGenLoader;
 import muramasa.gti.proxy.ClientHandler;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.TagsUpdatedEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.event.lifecycle.FMLClientSetupEvent;
 import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
 import net.minecraftforge.fml.javafmlmod.FMLJavaModLoadingContext;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
+
 @Mod(Ref.ID)
 public class GregTech extends AntimatterMod {
 
+    public static class WorldLoadListener {
+        @SubscribeEvent
+        public static void TagsUpdatedEvent(TagsUpdatedEvent event)
+        {
+            MaterialRecipeLoader.init();
+            MachineRecipeLoader.init();
+        }
+    }
```

Comment:
Not elegant, and would like to have this in Antimatter, but this will do for now.